frontend/src/agenda/agenda.vue

```diff
@@ -9,6 +9,9 @@
                    :events="events()"
                    :config="config"></full-calendar>
     <b-container fluid>
+      <b-alert show v-if="exams.length === 0" variant="primary">
+        There are no exam events to display.  <b>Note:</b> The Agenda only shows events scheduled on or after today's date.
```

Comment:
Is there a technical reason for this change? I would assume that the agenda should show, say, exams written on Monday when on the weekly view for a Wednesday.